GitUI/CommandsDialogs/CommitDialog/WordWrapper.cs

```diff
@@ -17,6 +18,7 @@ public static string Wrap(string text, int lineLimit)
 
             return string.Join(Environment.NewLine, newLines);
         }
+#endif
```

Comment:
mmm... what is this for?

Reply:
My feeling was that although the method isn't currently used, it could be useful one day. You could say that of all the removed code, but I felt this part in particular might come of use.

There's another example where I mask out some code using preprocessor directives in dd52bda14bd60cd04f06f29a7e55987772e142dd.

I'm happy to delete the code if you feel it should go. Just let me know.

Reply:
My take - if it isn't used - delete it. If one day we need it - we can restore it or write it afresh.
Less code = less bugs = less maintenance.

Reply:
Fair enough. I tend to think the same.

In the case of `DvcsGraph.TopoSort` there's a comment in the method:

> TopoSorting is an easy way to detect if something has gone wrong with the graph

Also:

> // Sanity check

> Debugger.Break();

> Debug.WriteLine("*** {0} *** {1} {2}", junction, Nodes.Count, _junctions.Count);

My assumption is that this method is useful when debugging changes to the graph. Perhaps it would be useful to move it to test code in future. The implementation is less trivial than much of the other code that's removed in this PR. Either way it should probably not be in the compiled binary.

Reply:
That's ok, let's keep it for now.